runtime/compiler/env/VMJ9.h

```diff
@@ -794,6 +794,41 @@ class TR_J9VMBase : public TR_FrontEnd
     *    VM access is not required
     */
    TR_OpaqueMethodBlock* targetMethodFromMethodHandle(TR::Compilation* comp, TR::KnownObjectTable::Index objIndex);
+
+   /**
+    * \brief
+    *    Get the signature For MethodHandle.linkToStatic call for unresolved invokehandle
+    *
+    *    For unresolved invokeHandle, we do not know the adapter method at
+    *    compile time. The call is expressed as a call to the signature-polymorphic
+    *    MethodHandle.linkToStatic method. In addition to the arguments of the original call,
+    *    we need to provide the target and appendix objects as the last two arguments, in addition
+    *    to the MethodHandle object as the first argument. Therefore, we need to modify the
+    *    signature of the original call and adapt it to accept three extra arguments.
+    *
+    * \param comp the current compilation
+    * \param romMethodSignature the ROM Method signature to be processed
+    * \return char * the signature for linkToStatic
+    */
+   char *                getSignatureForLinkToStaticForInvokeHandle(TR::Compilation* comp, J9UTF8* romMethodSignature);
```

Comment:
Just curious why there is so much whitespace preceding the function name.  It doesn't match the style of the surrounding code.